src/python/pants/backend/scala/dependency_inference/ScalaParser.scala

```diff
@@ -17,26 +20,77 @@ import scala.collection.SortedSet
 import scala.collection.mutable.{ArrayBuffer, HashMap, HashSet}
 import scala.reflect.NameTransformer
 
-case class AnImport(
+object Constants {
+  val RootPackageQualifier = "_root_"
```

Comment:
Totally minor, but is it worth referencing some docs/definition of this qualifier? e.g. https://docs.scala-lang.org/tour/packages-and-imports.html#imports